### DISCUSSION TITLE
Auxiliary opening button on intercom

### DISCUSSION THREAD

**Discussion by S12Link** [PricelessToolkit/ESPBell-LITE#26] [Q&A]
Hi. First of all, my English is very bad and I am using a translator, so I apologize in advance for not expressing myself well.
My intercom has two buttons for two doors, each button opens a different door (main and auxiliary).
My question is if espbell can be used to open the second door, activating the second auxiliary button.
And if this activation is possible, I would like to know if it can be done with the current programming or do I have to change the espbell lite code.
I attach images of the operation of my intercom.
Thank you very much in advance.
![Screenshot_20240907-192645~2](https://github.com/user-attachments/assets/ca639317-5875-4838-8191-22346bac49c3)
![Screenshot_20240907-192617~2](https://github.com/user-attachments/assets/98c852bc-758e-445b-8c18-2e3179ddb643)
![Screenshot_20240907-190508~2](https://github.com/user-attachments/assets/efb70fca-e8bf-4df4-a936-21571aaa454d)
![Captura desde 2024-09-07 19-33-26](https://github.com/user-attachments/assets/a37c2afd-8e05-4a95-ab5a-207d1d9be223)

Model intercom: T71Z
1Z - Bell
2 - GND
3 - Lock
4 - Micro
5 - Speaker
P - Aux.
P - Aux.

E- 30 module: voltage at output terminal 12V 1A (9-10 terminals)

**Comment by PricelessToolkit** [comment]
Hi, do you want to open 2 doors at the same time?

**Reply by S12Link** [reply]
Hi, The ideal situation is to be able to open each door with its button independently: one button for the main door and another button for the secondary door. 
But if this is currently complicated or impossible, opening both doors at the same time is a solution that seems acceptable to me and would be good too.

**Reply by PricelessToolkit** [reply]
The lock terminals on the ESPBell have two pairs of wires, so you can modify them to open two doors at the same time, although this isn't the intended use case. You can connect one pair to GND, then split the other pair into two and connect them to the AUX terminals. However, keep in mind that the SSR relay can handle up to 2.5A continuously and a 5A spike for 100ms. If the combined current of the two solenoids exceeds 2.5A or if the spike goes above 5A, the SSR could burn out.

It might be a better idea to purchase a 3.3-5V SSR board from Aliexpress, connect it to the ESPBell's available outputs, and add a few lines of code to make it work. If you're not experienced with Arduino or similar electronics, I wouldn't recommend trying this.

**Comment by S12Link** [comment]
The lock works with 880 mA, so current is not a problem (see image below)

![Screenshot_20240908-082356](https://github.com/user-attachments/assets/c773dbee-c9c3-4618-9f30-82d24a6bdc98)

For the operation of the auxiliary opening button, there is a relay that performs this function (see image below)

![Screenshot_20240908-082955](https://github.com/user-attachments/assets/dc8ff848-7a53-469b-810e-cfaf2ddb9ed0)

The connection diagram of this relay, which works activated by the auxiliary button, is what you can see in the image below

![365389439-efb70fca-e8bf-4df4-a936-21571aaa454d](https://github.com/user-attachments/assets/6a723b97-3958-4774-a1f0-3682468c6ed0)

Well, unfortunately I have two problems:
1.- The main lock does not open when you press the main button (a click sounds but the lock does not open)

2.- I don't understand how the second lock works with the auxiliary button.
If I measure with the multimeter between terminal 3 and P = 12V.
If I measure with the multimeter between terminal 3 and the other P = 12V.
Looking at the diagram of this last image, is it possible to activate it?
Thanks in advance.

*NOTE: The mobile notification does work, although I have had to remove the image for it to reach the smartphone (Is there a compatible image resolution?)

**Reply by PricelessToolkit** [reply]
Quick question: If you don't pick up the handset and just press the open button on the intercom, do the doors open?

Regarding the image, typically the resolution provided on GitHub should work. Perhaps it's not placed in the correct directory on Home Assistant? You might want to double-check that.

As for the connection issue, it's really difficult to diagnose remotely without being physically present. There are many factors that could be causing it.

**Reply by S12Link** [reply]
> Quick question: If you don't pick up the handset and just press the open button on the intercom, do the doors open?

Yes, open anyway (main and aux.)

>Regarding the image, typically the resolution provided on GitHub should work. Perhaps it's not placed in the correct directory on Home Assistant? You might want to double-check that.

Apologies for this, I misspelled the file path.

>As for the connection issue, it's really difficult to diagnose remotely without being physically present. There are many factors that could be causing it.

Ok, I will continue investigating. Thank you very much for your time friend.
If I get anything new, I'll come and tell it. 
Thanksssss

**Reply by S12Link** [reply]
Hello again,
Like I said, I've continued doing some research on this topic.

Since it does not work by making normal connections, I have tried connecting the relay output cables directly to the intercom button (images attached).

![IMG_20240928_184735](https://github.com/user-attachments/assets/55a0bab0-2d6b-40e9-9872-832e0ea08711)
![IMG_20240928_184744](https://github.com/user-attachments/assets/001fce7f-f439-4f90-af38-725c7bfb826a)

This way I can only open one door, but it opens correctly.
I think it is the only way for it to work, due to the installation of this intercom itself.

To open the second door, I ordered a dry contact relay on Aliexpress, powered by 5V and with Wi-Fi connectivity:

https://acesse.one/80EVH

Am I doing something crazy that could damage the intercom or is it all possible how I am doing it?

Rescheduling the espbell is something I don't dare to do, because I don't want the notifications to stop working, which is what I really need.

Opening the door is secondary, but as a good DIYer that I am, I can't stop looking for alternatives XD.

**Reply by PricelessToolkit** [reply]
`as a good DIYer that I am, I can't stop looking for alternatives XD.` 
it's the only way!

You have two separate opening buttons, right? The ESPBell-MAX already has two relays. Why do you need an additional relay?

**Reply by S12Link** [reply]
I have espbell lite. I didn't buy the MAX version because I didn't take into account the two openings and it didn't need a battery.
